src/transformers/pipelines.py

```diff
@@ -115,7 +125,7 @@ def __init__(self, task: str, model: str, reason: str):
 
 class ArgumentHandler(ABC):
     """
-    Base interface for handling varargs for each Pipeline
+    Base interface for handling varargs for each :class:`~transformers.pipelines.Pipeline`.
```

Comment:
Maybe update this `varargs` to `argument` as well as you've done in the `DefaultArgumentHandler`?
```suggestion
    Base interface for handling arguments for each :class:`~transformers.pipelines.Pipeline`.
```